### ISSUE TITLE
Error in common.js on post.php

### ISSUE THREAD

**Issue by hirasso** [qtranslate/qtranslate-xt#843]
Hi there, I just noticed a bug: When I register a custom post type and don't add support for either 'title' or 'editor' to it, qtranslate-xt trows a bug on the `post-new.php` and `post.php` screens for this post type:

```php
/* 
* Registers a post type that won't support any built-in fields like title or editor. 
* Throws error in qtranslate-xt's common.js.
*/
register_post_type('my-post-type', [
   'supports' => false, 
])
```
The error:
```
common.js?ver=3.8.1.1590158756:800 Uncaught ReferenceError: setTinyMceInit is not defined
    at qTranslateX.addCustomContentHooks (common.js?ver=3.8.1.1590158756:800)
    at HTMLDocument.<anonymous> (post-exec.js?ver=3.8.1.1589546730:107)
    at i (jquery.js?ver=1.12.4-wp:2)
    at Object.fireWith [as resolveWith] (jquery.js?ver=1.12.4-wp:2)
    at Function.ready (jquery.js?ver=1.12.4-wp:2)
    at HTMLDocument.J (jquery.js?ver=1.12.4-wp:2)
```

**Comment by herrvigg** [comment]
Good point, we should probably handle this when we manage the post types that appear in the advanced options. If there is no editor we should certainly ignore those. We can make a try like this.

**Comment by herrvigg** [comment]
But how do you get to this error? If i just open `post-new.php` or `post.php` i don't see it. If i add `post-new.php?post_type=my-post-type` the server responds with "Invalid post type" which makes sense since the post is not supported. So you may have something special behind this.

As a workaround it should be enough to uncheck the support of translations for that post type in this tab option (the first section appearing in the advanced options). I expect this post type to appear there.

**Comment by hirasso** [comment]
Hi @herrvigg ,

I register the post type in my theme's `functions.php`, like described [in the docs](https://developer.wordpress.org/reference/functions/register_post_type/). I use the `init` hook for the post type registration:

```php
add_action( 'init', 'my_cpt_init' );
function my_cpt_init() {
    register_post_type( ... );
}
```

**Comment by herrvigg** [comment]
But how do you get to a URL with `post-new.php` or `post.php` with that post type? If you can't edit i don't understand what WP is supposed to display. You have something else going on here that i don't see.

**Comment by herrvigg** [comment]
@hirasso the problem is that you seem to need qTranslate for translations in ACF but how is that suppposed to work if the common.js can't be initialized normally? This is quite a very special case to handle. You can try one thing: change edit mode to single language mode just to disable the LSB. Let me know how this works.

**Comment by hirasso** [comment]
@herrvigg My use case is as following: I have a post type 'person', that has a custom field 'first_name' and 'last name', the post_title is being generated from those values. I would like to still use qtranslate for other fields in this post type. I think just checking for the existence of `setTinyMceInit` in common.js would be totally enough to fix this, don't you think?

**Comment by herrvigg** [comment]
Maybe this function has a bad name but it is from qTranslate. It checks inside the function if there is the classic TinyMCE interface. If this function is not defined, there is something else that makes the `qTranslateConfig.ctx` object inconsistent. If you have LSB enabled, this function should be defined in any case so if just add a check in `addCustomContentHooks` i'm quite sure something else will break so it's better if we try to understand better what is going wrong. Can you double-check again all the errors you have in the console? Maybe there is another error appearing before the one you reported?

Just as a workaround, you can switch temporarily to Single Language Mode (no LSB). I think this should work.
